src/documents/WebGamesAndSimulations.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Web Games and Simulations
+layout: default
+category: Methods
+---
+
+Interactive games can provide rich learning experiences for many students - but for some, these games are a accessibility and inclusivity challenge. This guide aims at giving some practical techniques to help improve the inclusiveness and accessibility of interactive content whether retrofitting existing content or creating a new game from scratch.
```

Comment:
"a" should be "an" in "these games are a accessibility and inclusivity challenge"